cmd/kapacitor/main.go

```diff
@@ -661,11 +661,13 @@ Options:
 }
 
 func doDefine(args []string) error {
```

Comment:
Can you update the usage comment for the define command to indicate that the id is optional if a `-file` is provided?

Reply:
it's not optional. we agreed that it should match what's on the cmd line.  I'll update in that context, though.